addons/account/tests/test_invoice_taxes.py

```diff
@@ -432,3 +432,48 @@ def test_misc_journal_entry_tax_tags_purchase(self):
             {'balance': -100.0,     'tax_ids': [],              'tax_tag_ids': self.tax_tag_pos.ids},
             {'balance': 1100.0,     'tax_ids': [],              'tax_tag_ids': []},
         ])
+
+    def test_tax_calculation_foreign_currency_large_quantity(self):
+        ''' Test:
+        Foreign currency with rate of 1.1726 and tax of 21%
+        price_unit | Quantity  | Taxes
+        ------------------
+        2.82       | 20000     | 21% not incl
+        '''
+        self.env['res.currency.rate'].search([]).unlink()
+        self.currency_usd_id = self.env.ref("base.USD")
+        self.currency_chf_id = self.env.ref("base.CHF")
+        # make sure that we have USD as base...
+        self.cr.execute("UPDATE res_company SET currency_id = %s WHERE id = %s", 
+            [self.currency_usd_id.id, self.env.company.id])
+        self.env['res.currency.rate'].create({
+            'currency_id': self.currency_chf_id.id,
+            'rate': 1.1726,
+            'name': '2001-01-01'})
+        invoice = self.env['account.move'].create({
+            'type': 'out_invoice',
```

Comment:
```suggestion
            'move_type': 'out_invoice',
```

Reply:
I pushed the change, do you need to robodoo or can I do that on this one ?